SpokestackTests/TextToSpeechTest.swift

```diff
@@ -32,6 +32,17 @@ class TextToSpeechTest: XCTestCase {
         let config = SpeechConfiguration()
         let tts = TextToSpeech(delegate, configuration: config)
         
+        // bad input results in a failed request that calls failure
```

Comment:
You do cover this in the description...

Reply:
the comment is to break it up visually when scanning